error: Node `foo` is declared twice
 --> tests/fail/ui/causality/redef-node.rs:5:5
  |
5 | /     node foo() returns ();
6 | |     let tel;
  | |___________^
